test/go.mod

```
@@ -0,0 +1,239 @@
+module botkube.io/botube/test
+
+go 1.21.6
```

Can we keep just the major and minor versions, similarly to our main go.mod in this repo?
```suggestion
go 1.21
```